plugins/containerised-app-with-assets/.toolkitrc.yml

```diff
@@ -11,7 +11,19 @@ commands:
     - Webpack:
         envName: development
         watch: true
-  'deploy:review': [] # Coming soon, see https://financialtimes.atlassian.net/browse/CPREL-1280
+  'deploy:review':
+    - Webpack:
+        envName: production
+    - UploadAssetsToS3
+    - DockerAuthCloudsmith
+    - DockerBuild
+    - DockerPush
+    - AwsAssumeRole:
+        roleArn: !toolkit/option '@dotcom-tool-kit/containerised-app.awsRoleArnStaging'
```

Comment:
```suggestion
        roleArn: !toolkit/option '@dotcom-tool-kit/containerised-app-with-assets.awsRoleArnStaging'
```

Reply:
The `containerised-app-with-assets` plugin reuses the configurations from `containerised-app` so this is correct, see the other `!toolkit/option` lines further down this file. See [this commit](https://github.com/Financial-Times/dotcom-tool-kit/commit/4f6fac7e5a5bd93d515031d8eed5bffc3e2e5fa4) for the change and [this comment](https://github.com/Financial-Times/dotcom-tool-kit/pull/778#discussion_r1960010617) from @ivomurrell explaining why

Reply:
Ah ah cool thanks!